### PR TITLE
add polish language support

### PR DESCRIPTION
**Pull request by waltzaround**
Adds polish language support courtesy of @MattOfNZ's polish friend

### CODE REVIEW

File: src/translations/locales/common_pl-PL.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "subtitle": "Zobacz sposoby na zaszczepienie się w twojej okolicy.",
+    "disclaimerNotAGovWebsite": "To nie jest oficjalna strona rządu. Aby zaszczepić się odwiedź stronę <0>bookmyvaccine.nz</0>",
+    "loading": "Ładowanie...",
+    "getDirections": "Jak dojechać",
+    "makeABooking": "Zrób rezerwacje",
+    "kmAway": "{{distance}}km odległości",
+    "lastUpdated": "Last updated {{updatedAt}}",
+    "cancel": "Cancel",
+    "website": "Website",
+    "address": "Address"
```
Comment (olafwrieden) [inline src/translations/locales/common_pl-PL.json:14]: Translation required.

File: src/translations/locales/common_pl-PL.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      "50km": "W zasięgu 50km",
+      "100km": "W zasięgu 100km"
+    },
+    "locationModal": {
+      "locaitonTitle": "Location",
+      "locationCTA": "Type in an address or suburb",
+      "useCurrentLocation": "Use my current location",
+      "geolocationNotSupported": "Geolocation is not supported by your browser"
+    },
+    "unknownLocation": "Unknown Location"
```
Comment (olafwrieden) [inline src/translations/locales/common_pl-PL.json:37]: Translation required.

File: src/translations/locales/common_pl-PL.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    "hours": "Godziny",
+    "publicHolidays": "Święta państwowe",
+    "otherExeptions": "Wyjątki",
+    "seeMore": "Zobacz więcej",
+    "closed": "Closed",
+    "otherLocations": {
+      "sectionHeader": "Other Vaccination Locations <0>- Open Today</0>",
+      "disclaimer": {
+        "title": "This location has been submitted by the public",
+        "message": "The information here may not be 100% accurate. Please refer to the location's website."
```
Comment (olafwrieden) [inline src/translations/locales/common_pl-PL.json:63]: Translations required.

File: src/translations/locales/common_pl-PL.json [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        "stepTwo": "Naciśnij na <i>Zrób rezerwacje</i> guzik, który cię przeniesie do strony <0>bookmyvaccine.nz</0>",
+        "stepThree": "Wpisz Swoje dane.",
+        "stepFour": "Następnie odpowiedz na pytanie, które miejsce i czas ci odpowiada."
+      },
+      "availableSlots": "Dostępne miejsca",
+      "noBookingsAvailable": "Nie ma dostępnych rezerwacji w tym dniu :(",
+      "backToCalendar": "Powrót do kalendarza",
+      "notBMV": "This location does not use <0>bookmyvaccine.nz</0> and may not have empty slots."
+    },
+    "numberOfAppointments": "{{sumAvailableTimes}} available"
```
Comment (olafwrieden) [inline src/translations/locales/common_pl-PL.json:83]: Translations required.

File: src/translations/locales/common_pl-PL.json [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  },
+  "footer": {
+    "message": "Jeżeli ta strona pomogła ci, rozważ udostępnienie jej innym",
+    "links": {
+      "contactUs": "Zkontaktuj się z Nami",
+      "rawData": "Dane podstawowe",
+      "sourceCode": "Kod źródłowy",
+      "roadmap": "Mapa drogowa"
+    },
+    "addressFinderLinkCopy": "Thanks to <0>AddressFinder</0> for free-of-charge address lookup"
```
Comment (olafwrieden) [inline src/translations/locales/common_pl-PL.json:93]: Translation required.

**Review by olafwrieden — CHANGES_REQUESTED** [review]
Some missing translations but otherwise looking fine :)

**Comment by waltzaround** [comment]
its coming @olafwrieden

**Comment by noway** [comment]
@waltzaround have we got the changes

File: src/translations/locales/common_pl-PL.json (hunk 1 above)
Comment (waltzaround) [reply src/translations/locales/common_pl-PL.json]: just added

File: src/translations/locales/common_pl-PL.json (hunk 2 above)
Comment (waltzaround) [reply src/translations/locales/common_pl-PL.json]: added

File: src/translations/locales/common_pl-PL.json (hunk 3 above)
Comment (waltzaround) [reply src/translations/locales/common_pl-PL.json]: added

File: src/translations/locales/common_pl-PL.json (hunk 4 above)
Comment (waltzaround) [reply src/translations/locales/common_pl-PL.json]: added

File: src/translations/locales/common_pl-PL.json (hunk 5 above)
Comment (waltzaround) [reply src/translations/locales/common_pl-PL.json]: added

**Review by waltzaround — COMMENTED** [review]
just came in fresh

**Review by olafwrieden — APPROVED** [review]

**Review by noway — COMMENTED** [review]
LGTM

**Review by noway — COMMENTED** [review]
LGTM

**Pull request merged**